scrapy/utils/serialize.py

```diff
@@ -36,4 +38,10 @@ def default(self, o: Any) -> Any:
 
 
 class ScrapyJSONDecoder(json.JSONDecoder):
-    pass
+    def __init__(self, *args, **kwargs):
+        warnings.warn(
+            "'ScrapyJSONDecoder' is a deprecated setting.\nIt will be removed in a future version of Scrapy.",
```

Comment:
It's a class, not a setting.

Reply:
Should make the format for this warning(for the class) a two line statement just to distinguish it from a deprecation message for a function or make it a one-liner like a function?

Reply:
One-line please.